packages/prisma/schema.prisma

```diff
@@ -1379,3 +1380,15 @@ model PlatformBilling {
 
   team Team @relation(fields: [id], references: [id], onDelete: Cascade)
 }
+
+model AuditLog {
+  id         Int      @id @default(autoincrement())
+  action     String
+  actionDesc String?
+  actor      Json
+  target     Json?
```

Comment:
Same here on target. We can have a `targetId: 1` and `targetType: "team"` instead of a Json field.

Reply:
Unfortunately, We can't turn this in to primitive data type because there are different triggers. For Example Creation of event type only require targetEvent but Updating eventtype require changedAttribute for report. So We need to wrap it into target JSON.